smack-experimental/src/main/java/org/jivesoftware/smackx/httpfileupload/element/Slot.java

```diff
@@ -66,18 +51,15 @@ public URL getGetUrl() {
     }
 
     public Map<String, String> getHeaders() {
-        return headers;
+        return Collections.unmodifiableMap(Collections.<String, String>emptyMap());
     }
 
     @Override
     protected IQChildElementXmlStringBuilder getIQChildElementBuilder(IQChildElementXmlStringBuilder xml) {
         xml.rightAngleBracket();
 
-        xml.element("put", putUrl.toString());
-        xml.element("get", getUrl.toString());
-        for (Map.Entry<String, String> entry : getHeaders().entrySet()) {
```

Comment:
I have not seen this syntax in the XEPs at all. Afaik headers were only introduced in version 0.3 of the XEP (along with the namespace bump).